packages/grafana-ui/src/components/ColorPicker/SpectrumPalette.tsx

```diff
@@ -15,15 +15,27 @@ export interface SpectrumPaletteProps {
 
 const SpectrumPalette: React.FunctionComponent<SpectrumPaletteProps> = ({ color, onChange }) => {
   const [currentColor, setColor] = useState(color);
-  useThrottleFn(onChange, 500, [currentColor]);
+
+  const formatHexColor = (color: string) => {
+    const newColor = tinycolor(color);
+    return newColor.toString(newColor.getAlpha() === 1 ? 'hex6' : 'hex8');
+  };
```

Comment:
Could we move this function to colorManipulator and use it here and in fieldOverrides?